pages/api/id/[form]/retrieval.ts

```diff
@@ -46,6 +46,34 @@ function connectToDynamo(): DynamoDBDocumentClient {
   return DynamoDBDocumentClient.from(db);
 }
 
+interface dynamoDBItem {
+  FormID: { S: string };
+  SubmissionID: { S: string };
+  FormSubmission: { S: string };
+  [key: string]: Record<string, string>;
+}
+
+/**
+ * Parses DynamoDB Item into vault response object
+ * @param response DynamoDB Vault Item
+ * @returns Object representative of Item
+ */
+const parseToObject = (response: dynamoDBItem) => {
```

Comment:
Seems like this could be a one liner in the map function

Reply:
I had originally seperated it out for easier readability but I've moved it to be directly within the map function.